minecraft/dial.go

```diff
@@ -54,6 +54,10 @@ type Dialer struct {
 	// If TokenSource is nil, the connection will not use authentication.
 	TokenSource oauth2.TokenSource
 
+	// XBLToken should be used in place of TokenSource if the XBL token is already known, i.e through a different
+	// oauth source
+	MultiplayerToken *auth.XBLToken
```

Comment:
Doc name and field name don't match. I think calling the field XBLToken like the comment describes is probably less confusing.